### PR TITLE
feat: default project on validate

### PR DESCRIPTION
**Pull request by rephus**
Closes:  #5673

### Description:

image

### CODE REVIEW

File: packages/cli/src/handlers/validate.ts [hunk 1]
```diff
@@ -6,10 +6,10 @@ import {
     SchedulerJobStatus,
 } from '@lightdash/common';
 import columnify from 'columnify';
-import { getConfig } from '../../config';
-import GlobalState from '../../globalState';
-import * as styles from '../../styles';
-import { checkLightdashVersion, lightdashApi } from './apiClient';
+import { getConfig } from '../config';
```
Comment (rephus) [inline packages/cli/src/handlers/validate.ts:9]: I moved this validate out of the dbt folder

File: packages/cli/src/handlers/validate.ts [hunk 2]
```diff
@@ -63,26 +63,33 @@ export const validateHandler = async (options: ValidateHandlerOptions) => {
...
 
     if (options.project) {
         console.error(`Validating project ${projectUuid}\n`);
-    } else {
+    } else if (config.context?.previewProject) {
         console.error(
             `Validating preview project ${styles.bold(
                 config.context?.previewName,
-            )} with projectUuid ${config.context?.previewProject}\n`,
+            )} with projectUuid ${projectUuid}\n`,
```
Comment (TuringLovesDeathMetal) [inline packages/cli/src/handlers/validate.ts:89]: Can we change this to the project name? 

So like: `Validating preview project **Humble Bumblebee**` (where the project name is bold)

Or `Validating project **Lightdash Analytics**`


^I know that projects can have the same name, but it's really unlikely for them to be named the same and I feel like the name is more intuitive/helpful than the UUID

**Review by TuringLovesDeathMetal — CHANGES_REQUESTED** [review]
This looks really great! Just one small change for replacing teh UUID to the project name in one fo the messages :)

File: packages/cli/src/handlers/validate.ts (hunk 2 above)
Comment (rephus) [reply packages/cli/src/handlers/validate.ts:89]: at the moment we don't store the project name, but I can add it , let me change this

**Review by TuringLovesDeathMetal — APPROVED** [review]
nice!

**Pull request merged**